funding/manager_test.go

```diff
@@ -4253,3 +4253,36 @@ func TestFundingManagerZeroConf(t *testing.T) {
 	// have been deleted from the database, as the channel is announced.
 	assertNoFwdingPolicy(t, alice, bob, fundingOp)
 }
+
+// TestCommitmentTypeFundmaxSanityCheck was introduced as a way of
```

Comment:
nit: comments should try to fill the 80 characters where possible, to make things look more uniform and less jagged.